### PR TITLE
Integrate gutenberg-mobile release 1.59.2

### PR DESCRIPTION
**Pull request by fluiddot**
## Description
This PR incorporates the 1.59.2 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3844

Release Submission Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
     ext.wordPressUtilsVersion = 'develop-f98c841225fbc61031f32426aaef4e347b34f717'
     ext.wordPressLoginVersion = '0.0.3'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.59.1'
+    ext.gutenbergMobileVersion = '3844-8992fef55c6991b7f24651f3ac65ac11c38550f2'
```
Comment (dcalhoun) [inline build.gradle]: Before merging, we'll need to update the `gutenbergMobileVersion` ref to the release tag once it is created.

**Review by dcalhoun — APPROVED** [review]
LGTM. Left one comment merely as a reminder. 

My understanding is the bug has been present for awhile. Because of that, we may should mention in the fix in the release notes. I believe it is too late for the note to be published to the Play Store, but it might still be beneficial to include the note for posterity. WDYT?

**Comment by twstokes** [comment]
@fluiddot I ran through the Writing Flow Checklist for the above WordPress APK and all looked well to me. 👍

**Comment by AliSoftware** [comment]
This has now been shipped as part of the new betas for WordPress and Jetpack `18.0-rc-3` 🎉

**Pull request merged**